src/wireframes/components/properties/CustomSlider.tsx

```diff
@@ -23,18 +23,11 @@ interface CustomSliderProps {
 }
 
 export const CustomSlider = React.memo(({ max, min, onChange, value }: CustomSliderProps) => {
-    const [sliderValue, setSliderValue] = React.useState<number>(value);
```

Comment:
My code is weird, are you sure that there is not a good reason for that? :D

Reply:
Well, I did not think your code was weird. It just does something different. Your applies the change when the slider is let go. Mine applies the change as soon as the slider is moved. So it comes to a question of UI preferences.

I like the apply immediately style better. What is your preference?

I just realized that the undo treats every little step as separate undo step. I think that would need to be changed. I'll split PRs..